clients/drcachesim/tests/burst_syscall_inject.cpp

```diff
@@ -248,12 +270,15 @@ postprocess(void *dr_context, std::string syscall_trace_template_file,
 }
 
 static basic_counts_t::counters_t
-get_basic_counts(const std::string &trace_dir)
+get_basic_counts(const std::string &trace_dir, syscall_mix_t::statistics_t &syscall_stats)
```

Comment:
(Optional) May be nicer to just return basic_counts_t::counters_t also in a new reference param (for consistency with syscall_stats), and have get_basic_counts return void.

Reply:
Done, and renamed to get_tool_results().